src/java.desktop/macosx/native/libawt_lwawt/awt/a11y/CommonComponentAccessibility.m

```diff
@@ -126,7 +126,7 @@ + (void) initializeRolesMap {
     /*
      * Here we should keep all the mapping between the accessibility roles and implementing classes
      */
-    rolesMap = [[NSMutableDictionary alloc] initWithCapacity:51];
```

Comment:
I suppose 51 just seemed like a strange number ?

Reply:
I removed one position from this dictionary, and therefore reduced the capacity.